.gitignore

```diff
@@ -2,12 +2,14 @@ __pycache__
 .idea
 .vscode
 .ninja_*
+.mypy_cache
 *.exe
 build
 build.ninja
 objdiff.json
 orig/*/*
 !orig/*/.gitkeep
-tools/mwcc_compiler/*
-!tools/mwcc_compiler/.gitkeep
```

Comment:
Should these lines be removed?

Reply:
It all gets downloaded into `build/tools` and `build/compilers` now and all of build gets ignored